playground/nested-deps/vite.config.js

```diff
@@ -9,6 +22,7 @@ export default defineConfig({
       '@vitejs/test-package-c/side',
       '@vitejs/test-package-d    > @vitejs/test-package-d-nested',
       '@vitejs/test-package-e > @vitejs/test-package-e-included',
+      packageFPath,
```

Comment:
@patak-dev do we want to support absolute paths passed to `optimizeDeps.include`? Maybe it does work now, but I'm not sure if this is an API usage we want to guarantee.

Reply:
Ya... thanks for flagging this. Simplified the test and it still works locally